snap/info_snap_yaml.go

```diff
@@ -364,6 +365,7 @@ func setAppsFromSnapYaml(y snapYaml, snap *Info, strk *scopedTracker) error {
 			Completer:       yApp.Completer,
 			StopMode:        yApp.StopMode,
 			RefreshMode:     yApp.RefreshMode,
+			InstallMode:     yApp.InstallMode,
```

Comment:
we need validation code in ValidateApp like for RefreshMode,  we should allow for "enable" and "disable", where "enable" is the default

Reply:
This is added now.